daemon/daemon.go

```diff
@@ -1486,13 +1485,11 @@ func isBridgeNetworkDisabled(conf *config.Config) bool {
 }
 
 func (daemon *Daemon) networkOptions(conf *config.Config, pg plugingetter.PluginGetter, hostID string, activeSandboxes map[string]interface{}) ([]nwconfig.Option, error) {
-	dd := runconfig.DefaultDaemonNetworkMode()
-
 	options := []nwconfig.Option{
 		nwconfig.OptionDataDir(conf.Root),
 		nwconfig.OptionExecRoot(conf.GetExecRoot()),
-		nwconfig.OptionDefaultDriver(string(dd)),
-		nwconfig.OptionDefaultNetwork(dd.NetworkName()),
+		nwconfig.OptionDefaultDriver(network.DefaultNetwork),
+		nwconfig.OptionDefaultNetwork(network.DefaultNetwork),
```

Comment:
As a follow-up, we should kill these options. Both are used in a single place, and both use don't make sense.

Reply:
Agreed. Or at least; we need to have better setting up of defaults. I think it's OK to inject defaults, but perhaps these should actually come from the daemon config (although I don't think the "default" for networks would ever be _configurable_ - always hard-coded depending on which platform we're running on).